packages/picongpu/package.py

```diff
@@ -79,7 +79,7 @@ class Picongpu(Package):
     depends_on('boost@1.65.1:1.70.0 cxxstd=11', when='backend=cuda ^cuda@9:')
     # note: NOT cuda aware!
     # depends_on('mpi@2.3:', type=['link', 'run'])
-    depends_on('openmpi@1.8:3.99', type=['link', 'run'])
+    depends_on('openmpi@1.8:3.99,4.0.1:', type=['link', 'run'])
```

Comment:
actually we can also update to the documented line above, so that this works with MPICH et al., too.

Reply:
DO you like to enable the line `depends_on('mpi@2.3:', type=['link', 'run'])`?

Reply:
Yeah, that should cut it. We could always add a `conflict` for the broken MPI versions.

I think asking for `mpi@3.0:` is safer, not sure if we are testing ancient MPIs that do only support 2.3 anymore :D

Reply:
info: PIConGPU is currently only checking the MPI version shipped by Ubuntu 18.04. It is `openmpi 2.1.1-8` which is MPI 3.1.